blog/2021/07/general-survey.md

```diff
@@ -78,53 +78,57 @@ While over 96% of packages in the General registry have licenses approved by the
 packages appear to not have any license at all, while 20 have a license which
 isn't automatically detected as OSI-approved.
 
-![Distribution of packages with OSI-approved licenses](/_assets/blog/2021-07-general-survey/osi-licenses.png)
+![Distribution of packages with OSI-approved licenses](/assets/blog/2021-07-general-survey/osi-licenses.png)
 
-The General registry was actually always supposed to include only packages with
-OSI-approved licenses, but this requirement has been automatically enforced only
-since last March: any new package or version of already registered packages must
-now have an OSI-approved license.  Since the introduction of this automated
+Julia's public package registries have long required OSI-approved licenses
+(stretching back to [October 2015](https://github.com/JuliaLang/METADATA.jl/pull/3788)),
+but this requirement was [not documented in the General registry README](https://github.com/JuliaRegistries/General/issues/24038)
+for some time, and has only been
+[automatically enforced since last March](https://discourse.julialang.org/t/fyi-the-general-registry-now-checks-for-an-osi-approved-software-license-when-registering-new-packages-and-new-versions/57018).
+Now, any new package or new version of an already-registered package must have
+an auto-detected OSI-approved license to get automatic approval for merging into
+General.  Since the introduction of this automated
```

Comment:
this is the biggest content change; I think <https://discourse.julialang.org/t/fyi-the-general-registry-now-checks-for-an-osi-approved-software-license-when-registering-new-packages-and-new-versions/57018> was basically the "official announcement" of the license check so I figured I'd link it, especially since it has more links and FAQ items than we'd want to put here.